TaskList/app/views/layouts/_errors.html.erb

```diff
@@ -0,0 +1,12 @@
+<% if @task.errors.any? %>
```

Comment:
This error messages partial is great. Again, we'll be seeing a lot more of this when we get into the ActiveRecord model validations stuff.